presto-main/src/main/java/com/facebook/presto/server/protocol/StatementResource.java

```diff
@@ -192,9 +193,10 @@ private static Response toResponse(Query query, QueryResults queryResults)
     {
         ResponseBuilder response = Response.ok(queryResults);
 
-        // add set catalog and schema
+        // add set catalog, schema, and path
```

Comment:
I would just remove this comment. It is trivial and code is self explaining here.